code/style.css

```diff
@@ -1,4 +1,201 @@
+*, *::before, *::after { box-sizing: inherit;}
+
 html {
-  background: #0025ff;
-  color: #fff;
+    display: flex;
+    background: #7e7171; 
+    color: #fff;
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+    header { 
+        display: flex;
+        background-color: black;
+        border: 5px outset red;
+        align-items: center;
+        justify-content: space-between;
+        border-radius: 10px;
+        width: 100%;
+    }
+
+    .logo {
+        display: flex;
+        border: 10px solid white;
+        background-color: white;
+        height: 100%;
+        align-items: center;
+        justify-content:center;
+        flex-direction: column;
+    }
+
+    .header-child {
+        margin: 5px;
+        width: 20%;
+        height: 80%;
+    }
+
+    .titel {
+        display: flex;
+        font-size: 100%;
+        text-align: center;
+        width: 100%;
+        height:100%;
+        color: black;
+        font-size: initial;
+    }
+
+    .navigation{
+          display: none;
+    }
+
+      .technico {
+            background-color: #d89191;
+            text-align: center;
+            height:  100%;
+            width: 100%; 
+      }
+
+      .zebra {
+          height:100%;
+          width: 100%;
+          border: 5px solid white;
+       }
+
+      .small-news {
```

Comment:
It is much clearer for the viewer to understand and follow the code if you put everything related to small news under big news as per your structure on your html according to best practice.